testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/ssl/TrustStoreEmailTest.java

```diff
@@ -85,14 +84,20 @@ public void afterTrustStoreEmailTest() {
         SslMailServer.stop();
     }
 
-    @Test
-    public void verifyEmailWithSslEnabled() {
+    public void verifyEmailWithSslEnabled(Boolean opportunistic) {
```

Comment:
Thank you @redabourial Everything looks good, just one minor change - can this be boolean instead as null isn't used.